md/project_deploy_in_studio.md

```diff
@@ -0,0 +1,36 @@
+# Project deployment in a Studio
+
+This page presents an overview of the _Deploy wizard_ capabilities in the Studio
```

Comment:
```suggestion
This page presents an overview of the _Deploy wizard_ capabilities in Bonita Development Suite.
```

Reply:
or "This page presents an overview of the _Deploy wizard_ capabilities that allow the deployment of a project, or parts of a project, from Bonita Studio to the embedded Tomcat."